### DISCUSSION TITLE
[Projects Beta] Add function to load all issues from repo

### DISCUSSION THREAD

**Discussion by mobile-sergey** [community/community#7644] [Projects and Issues] [answered]
I want to add all issues to one "project beta" from 4 repo (about 1000 issues)
How I can do it? I know only one method: do it by hands.
Can you add method: add all issues from one repo to project?
And can you add settings to add new issues to project automatically?

**Comment by mobile-sergey** [comment, marked as answer]
Find andswer to my question:

[Automatically add new issues from a repository into projects](https://github.com/github/roadmap/issues/286) - in road map

Other discussions about it:

- [Automatically add items from repositories (projects beta) #5378](https://github.com/github/feedback/discussions/5378) - automation in actions to add issues from repo to [Project Beta]

- [Auto add new issues to a project #5713](https://github.com/github/feedback/discussions/5713)

- [[Projects Beta] Make it easier to add issues to projects #6833](https://github.com/github/feedback/discussions/6833)

- [Projects (beta): Make it easier to add issues / convert existing projects #5493](https://github.com/github/feedback/discussions/5493)

**Reply by steebchen** [reply]
Cool – I'm wondering if this would also enable adding new issues to a team board (in contrast to a repository board), for all or selected org repos.

**Comment by joachimweyl** [comment]
This only adds new items what about all the items already in the repo?
